config/routes.rb

```diff
@@ -11,6 +11,7 @@
   post "/feedback" => "feedback#create"
 
   match "/auth/:provider/callback" => "auth#oauth_callback", via: [:get]
+  get "/auth/failure" => "auth#failure"
```

Comment:
I would be happy to mock this up, if I knew what series of events would lead to hitting this page. Right now my mock would like like:

```ruby
post "/auth/myusa"
# ignore response
get "/auth/failure"
```

which would not really tell us much more than that the route works, not what triggers its.